ticdc/ticdc-debezium.md

```diff
@@ -0,0 +1,138 @@
+            {
+                "type":"struct",
+                "optional":true,
+                "name":"default.test.t2.Value",
+                "field":"before",
+                "fields":[
+                    {
+                        "type":"int32",
+                        "optional":false,
+                        "field":"a"
+                    },
+                    {
+                        "type":"int32",
+                        "optional":true,
+                        "field":"b"
+                    }
+                ]
+            },
+            {
+                "type":"struct",
+                "optional":true,
+                "name":"default.test.t2.Value",
+                "field":"after",
+                "fields":[
+                    {
+                        "type":"int32",
+                        "optional":false,
+                        "field":"a"
+                    },
+                    {
+                        "type":"int32",
+                        "optional":true,
+                        "field":"b"
+                    }
+                ]
+            },
+            {
+                "type":"string",
+                "optional":false,
+                "field":"op"
+            },
+            ...
+        }
+    }
+}
+```
+
+The key fields of the preceding JSON data are explained as follows:
+
+| Field      | Type   | Description                                            |
+|:----------|:-------|:-------------------------------------------------------|
+| payload.op        | String | The type of the change event. `"c"` indicates an `INSERT` event, `"u"` indicates an `UPDATE` event, and `"d"` indicates a `DELETE` event.  |
+| payload.ts_ms     | Number | The timestamp (in milliseconds) when TiCDC generates this message. |
+| payload.before    | JSON   | The data value before the event statement change. For `"c"` events, the value of the `before` field is `null`.     |
+| payload.after     | JSON   | The data value after the event statement change. For `"d"` events, the value of the `after` field is `null`.     |
+| payload.source.commit_ts     | Number  | The `CommitTs` identifier when TiCDC generates this message.       |
+| payload.source.db     | String   | The name of the database where the event occurs.    |
+| payload.source.table     | String  |  The name of the data table where the event occurs.   |
+| schema.fields     | JSON   |  The type information of each field in the payload, including the schema information of the row data before and after the change.   |
+
+### Data type mapping
+
+The data format mapping in the TiCDC Debezium message basically follows the [Debezium data type mapping rules](https://debezium.io/documentation/reference/2.4/connectors/mysql.html#mysql-data-types), which is generally consistent with the native message of the Debezium Connector. However, for some data types, the following differences exist between TiCDC Debezium and Debezium Connector messages:
```

Comment:
I would suggest making it clear that we are following the Debezium type mappings used for MySQL compatibility. E.g. changing the first sentence to something like:

> The data format mapping in the TiCDC Debezium message basically follows the [Debezium data type mapping rules](https://debezium.io/documentation/reference/2.4/connectors/mysql.html#mysql-data-types), which is generally consistent with the native message of the Debezium Connector for MySQL.